lib/service/Service.ts

```diff
@@ -223,6 +225,12 @@ class Service extends EventEmitter {
 
     const { base, quote, rate } = this.getPair(pairId);
 
+    const chainConfig = this.getChainConfig(base);
+
+    if (!!chainConfig) {
```

Comment:
I would prefer `chainConfig !== undefined`. This style sill confuses me every time I read it.